MainModule/Server/Plugins/WebPanel.lua

```diff
@@ -241,13 +241,9 @@ return function()
 			local code, msg = res.StatusCode, res.StatusMessage
 
 			if code ~= 524 then
-				if data and data.message then
-					error("WebPanel: "..data.message)
-				end
-				server.Logs:AddLog("Script", "WebPanel Polling Error: "..msg.." ("..code..")")
-				break
+				print("WebPanel: Server Timeout")
```

Comment:
You should replace the print line with the old server.Logs:AddLog("Script", "WebPanel Polling Error: "..msg.." ("..code..")") line.